### PR TITLE
[13.0][ADD] account_check_deposit_operating_unit

### PR DESCRIPTION
**Pull request by newtratip**
This module add operating unit in check deposit.

### CODE REVIEW

File: account_check_deposit_operating_unit/models/check_deposit.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class AccountCheckDeposit(models.Model):
+    _inherit = "account.check.deposit"
+
+    operating_unit_id = fields.Many2one(
```
Comment (kittiu) [inline account_check_deposit_operating_unit/models/check_deposit.py:10]: I think you should have index here.
Without it, using security rule when we have big number of records will be slow.

File: account_check_deposit_operating_unit/security/check_deposit_security.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+            ref="account_check_deposit.model_account_check_deposit"
+        />
+        <field
+            name="domain_force"
+        >['|',('operating_unit_id','=',False), ('operating_unit_id','in', user.operating_unit_ids.ids)]</field>
+        <field name="name">Check Deposit from allowed operating units</field>
+        <field name="global" eval="True" />
+        <field eval="0" name="perm_unlink" />
+        <field eval="0" name="perm_write" />
+        <field eval="1" name="perm_read" />
```
Comment (kittiu) [inline account_check_deposit_operating_unit/security/check_deposit_security.xml:14]: Readonly is enough?

File: account_check_deposit_operating_unit/models/check_deposit.py (hunk 1 above)
Comment (newtratip) [reply account_check_deposit_operating_unit/models/check_deposit.py:10]: Yes, I agree with you.

Edited.

File: account_check_deposit_operating_unit/security/check_deposit_security.xml (hunk 2 above)
Comment (newtratip) [reply account_check_deposit_operating_unit/security/check_deposit_security.xml:14]: Enough, This is just rule for check operating unit.

Comment (kittiu) [reply account_check_deposit_operating_unit/security/check_deposit_security.xml:14]: I mean, user under this rule can create/edit document?

Comment (newtratip) [reply account_check_deposit_operating_unit/security/check_deposit_security.xml:14]: Yes, can create/edit document
Selection_451

**Review by alan196 — APPROVED** [review]
Technical Review 👍

**Review by kittiu — APPROVED** [review]

**Comment by newtratip** [comment]
@OCA/accounting-maintainers Should we merge this ?

**Comment by newtratip** [comment]
@OCA/accounting-maintainers, 
This pr is add operating unit in check deposit module. Is there any PSC member review/merge this ?

**Review by AaronHForgeFlow — APPROVED** [review]

**Comment by AaronHForgeFlow** [comment]
could you squash the two commits into one?

**Comment by newtratip** [comment]
@AaronHForgeFlow, done

**Pull request merged**